package.json

```diff
@@ -33,7 +33,16 @@
   "dependencies": {
     "match-sorter": "^6.3.0",
     "react-query": "^3.12.2",
-    "vue": "^3.0.7"
+    "vue-demi": "^0.7.4"
+  },
+  "peerDependencies": {
+    "@vue/composition-api": "^1.0.0-beta.1",
+    "vue": "^2.0.0 || >=3.0.0-rc.0"
```

Comment:
Since Vue 3 is already released and considered stable, I would prefer to add at least Vue 3.0.0 as a dependency.